packages/contracts/README.md

```diff
@@ -84,5 +84,39 @@ You can also build specific components as follows:
 yarn build:contracts
 ```
 
+### Deploying the Contracts
+To deploy the contracts first clone, install, and build the contracts package.
+
+Next set the following env vars:
+
+```bash
+CONTRACTS_TARGET_NETWORK=...
+CONTRACTS_DEPLOYER_KEY=...
+CONTRACTS_RPC_URL=...
+```
+
+Then to perform the actual deployment run:
+
+```bash
+npx hardhat deploy \
+  --ovm-address-manager-owner ... \
```

Comment:
I also think you should document all *required* parameters here.

Reply:
Do you mean I should have a separate section for required params? Or do you mean I should remove the optional params?

What I documented does include all required params I believe.